src/contexts/WorkDataContext.js

```diff
@@ -0,0 +1,56 @@
+import { createContext, useCallback, useState } from 'react';
+import { DateTime } from 'luxon';
+
+export const WorkDataContext = createContext();
+
+const createFormDate = () => {
+  return new DateTime.utc();
+};
+
+const createWorkValues = () =>
+  new Map(
+    Array(7)
+      .fill(createFormDate())
+      .map((date, index) => {
+        let nextDate = date.plus({ days: index });
```

Comment:
This is not a let.  We do not modify it.  Also this can be simplified to:
`.map((date,days) => date.plus({ days })`